modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtInstancelevel.testcase

```diff
@@ -136,6 +136,8 @@ definition {
 		}
 
 		task ("When exporting file in JSON format with all fields selected") {
+			ImportExport.mapExport(exportFields = "Attribute Code");
```

Comment:
the problem with this test is that a CSV is being donwloaded in zip instead of JSON, so I think even if we fix it it will fail. @Hijky can you confirm on your local what zip file is being downloaded? Yesterday for me it was CSV